update docs (#40)
aws-samples/d2l-java: README.md

This project is modified from the original [Dive Into Deep Learning](https://d2l.ai) book by Aston Zhang, Zachary C. Lipton, Mu Li, Alex J. Smola and all the community contributors. 
GitHub of the original book: [https://github.com/d2l-ai/d2l-en](https://github.com/d2l-ai/d2l-en). 

> We have modified the book to provide implementation in Java using [Deep Java Library(DJL)](https://djl.ai).

Reviewer: ```suggestion
We have adapted the book to to use Java and the [Deep Java Library(DJL)](https://djl.ai).
```

All the notebook here can be downloaded and run using Java Kernel. We also compiled the book into a [website](https://d2l.djl.ai).

> This project is currently been developed and maintained by the [DJL community](https://github.com/awslabs/djl).

Reviewer: ```suggestion
This project is currently being developed and maintained by AWS and the [DJL community](https://github.com/awslabs/djl).
```
